science/openmpi/Portfile

```diff
@@ -210,9 +209,9 @@ if {${subport} != ${name}} {
         append long_description " (AND THE FORTRAN COMPILER SELECTED BY THE VARIANT, IF ANY)"
 
         compilers.choose   fc
-        compilers.setup    default_fortran
```

Comment:
Should you also move the first choose line here too ?

Reply:
Sure.

Reply:
thanks